lib/redmine_required_field/patches/issues_controller_patch.rb

```diff
@@ -46,7 +70,7 @@ def rrf_params_match_settings(rrf_setting)
 
         setting_issue_status_ids = rrf_setting[RedmineRequiredField::ISSUE_STATUS_IDS]
         matched = RedmineRequiredField.setting_values_match_id(setting_issue_status_ids, params[:issue][:status_id])
-
+        
```

Comment:
trailing whitespace ?

Reply:
whitespaces removed. updated in next commit